libs/ojsonnet/Unit_jsonnet_subst.ml

```diff
@@ -3,6 +3,7 @@ open File.Operators
 module Y = Yojson.Basic
 
 let dir_pass = Fpath.v "tests/jsonnet/eval_pass"
```

Comment:
At this point you can use a list of directories, like
let dir_pass = [
  "tests/jsonnet/eval_pass";
  "tests/jsonnet/tutorial/pass"
  ...
 ]  |> File.Path.of_strings